src/robotide/editor/macroeditors.py

```diff
@@ -85,17 +91,38 @@ def delete_rows(self):
     def delete(self):
         self.kweditor.OnDelete()
 
-    def comment(self):
+    def comment_rows(self):
         self.kweditor.OnCommentRows()
 
-    def uncomment(self):
+    def uncomment_rows(self):
         self.kweditor.OnUncommentRows()
 
+    def move_cursor_down(self):
+        self.kweditor.OnMoveCursorDown()
+
+    def move_rows_up(self):
+        self.kweditor.OnMoveRowsUp()
+
+    def move_rows_down(self):
+        self.kweditor.OnMoveRowsDown()
+
+    def swap_row_up(self):
+        self.kweditor.OnSwapRowUp()
+
     def comment_cells(self):
-        self.kweditor.OnCommentCell(None)
+        self.kweditor.OnCommentCells(None)
 
     def uncomment_cells(self):
-        self.kweditor.OnUncommentCell(None)
+        self.kweditor.OnUncommentCells(None)
+
+    def make_variable(self):
```

Comment:
These functions repeat the action. For example, you'll get ${${}} .

Reply:
I think you added it in singular but I am not sure.